events/triggers/mount-fail

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+echo "Warning: external drive mount change detected, stopping Umbrel..."
+systemctl stop umbrel-external-storage
```

Comment:
I included an `external-storage` stop as well to hopefully kill any background processes that service spawned (<- not sure if that's how this works)